client/app/src/main/java/com/microsoft/research/karya/ui/dashboard/DashboardUiState.kt

```diff
@@ -3,7 +3,7 @@ package com.microsoft.research.karya.ui.dashboard
 import com.microsoft.research.karya.data.model.karya.modelsExtra.TaskInfo
 
 sealed class DashboardUiState {
-  data class Success(val data: List<TaskInfo>) : DashboardUiState()
+  data class Success(val taskInfoData: List<TaskInfo>, val totalCreditsEarned: Float?) : DashboardUiState()
```

Comment:
Instead of passing 2 parameters, make a class object.

```
data class DashboardState(taskInfoList: List<TaskInfo>, creditsEarned: Float = 0.0f)
```